docker-compose.yml

```diff
@@ -6,6 +6,9 @@ services:
     image: wurstmeister/zookeeper
     ports:
       - "2181:2181"
+    logging:
+      options:
+        max-size: 50m
```

Comment:
can we update to this format below (according to this logging options:https://docs.docker.com/compose/compose-file/compose-file-v3/):
```
logging:
      driver: "json-file"
      options:
        max-size: "10m"
        max-file: 5"
```

Reply:
The default for driver is "json-file", but I've added it in to avoid ambiguity. I also decreased the max file size to 10m and set the files to 5 as requested.

Reply:
Thank you!